docs/api/v1.md

```diff
@@ -86,18 +269,21 @@ to stream trace with chunked encoding, maybe omit status).
 ```
 
 
+
+
 ### File
 ```
 {
     name {string}: Name of file
     path {string}: Path to file
     type {string}: File type ("file" or "directory")
     contents: {string|string[]}:
-        If type="file", contents is a string of the base64 encoded file contents
-        If type="directory", contents is an array of File names inside the directory
+        If type="file", a string of the base64 encoded file contents
```

Comment:
Could we avoid base64ing files and just download them out-of-band, like a static file server?